data/mods/Aftershock/player/professions.json

```
@@ -434,5 +434,31 @@
     "traits": [ "PROF_MED", "UPLIFTED", "THRESH_CEPHALOPOD", "CEPH_EYES", "GILLS", "INT_UP_2", "LEG_TENTACLES", "SLIMY" ],
     "//": "Need to add underwater gear for this.  See extended leg_tentacles mutation.",
     "items": { "both": { "items": [ "wetsuit_cecalia", "trident", "dive_bag" ] } }
+  },
+  {
+    "type": "profession",
+    "id": "robot_scavenger",
+    "name": "Robot Scavenger",
+    "description": "Before the cataclysm struck, you were one of the members of an underground group striving to free cyborgs from their shackles, raiding laboratories and underground clinics in your pursuit of liberation.  Now that most of the humanity has been turned into zombies, your goal has become even more important to you.",
```

Thanks for this! need to bring in the description in line with the current lore for aftershock, but otherwise, it works perfectly fine!
```suggestion
    "description": "Scavenging and reselling abandoned robotics is what you call your "job." It certainly isn't official in any means of the word, but it rakes in the dough and that is all that matters to you, even if it involves dangerous activities, be it cracking old Wraitheon security cyphers or crossing well-worn wires on ancient mining drones.",
```